app/grandchallenge/components/form_fields.py

```diff
@@ -144,7 +145,12 @@ def get_json_field(self):
             JSONValidator(schema=default_schema),
             JSONValidator(schema=self.instance.schema),
         ]
+
         extra_help = ""
+
+        if self.instance.default_field == forms.JSONField:
+            extra_help = self.example_download_link
```

Comment:
```suggestion
        if self.instance.default_field == forms.JSONField:
            extra_help = self.example_download_link
        else:
            extra_help = ""
```

Sorry! Nitpick code styling.

Reply:
Done